site2/docs/admin-api-namespaces.md

```diff
@@ -417,7 +417,7 @@ admin.namespaces().getNamespaceMessageTTL(namespace)
 
 #### split bundle
```

Comment:
```suggestion
#### Split bundle
```

Reply:
FYI this capitalization is not consistent with the remainder of the document - if you agree I'm happy to make the rest of the document consistent with your proposed capitalization scheme. Thanks!

Reply:
Yes, we use sentence case style for headings in Pulsar docs.
Thank you very much for your contribution.

Reply:
I notice that the capitalization is not consistent, if you could refine the rest of them, that's really great.